### PR TITLE
Add warning about leaving migation labels.

### PR DESCRIPTION
**Pull request by katre**

### CODE REVIEW

File: docs/release-playbook.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ In the below, _X.Y_ is a release you are cutting.
 
 1. Search for issues with labels "migration-_X.Y-1_" that are not "breaking-change-_X.Y_"
 2. For all such issues, apply "migration-_X.Y_" label.
+   1. Do **not** remove any previous "migration-_X.Y-1_" labels.
```
Comment (fweikert) [inline docs/release-playbook.md]: I would move it to the same line as 2. (having a sublist with just one entry seems wrong).

Comment (katre) [reply docs/release-playbook.md]: Done.

**Comment by fweikert** [comment]
LGTM!

**Comment by katre** [comment]
I don't have permissions to merge, can you?

**Pull request merged**